cli/src/main.rs

```diff
@@ -216,12 +222,16 @@ fn override_configuration(mut config: Config, options: &ArgMatches) -> Config {
     }
 
     #[cfg(feature = "go")]
-    if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {
-        config.go.package = go_package.to_string();
+    {
+        if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {
+            config.go.package = go_package.to_string();
+        } else {
+            config.go.package = is_go_package_present(&config)?;
+        }
```

Comment:
prefer match over if else
```suggestion
        match options.value_of(args::ARG_GO_PACKAGE) {
            Some(go_package) => {
                config.go.package = go_package.to_string();
            },
            None => {
                config.go.package = is_go_package_present(&config)?;
            }
        }
```

Reply:
Done!